lib/bolt/bolt_option_parser.rb

```diff
@@ -444,10 +444,7 @@ def get_help_text(subcommand, action = nil)
           bolt project migrate [options]
 
       DESCRIPTION
-          Migrate a Bolt project to the latest version.
-
-          Loads a Bolt project's inventory file and migrates it to the latest version. The
-          inventory file is modified in place and will not preserve comments or formatting.
+          Migrate a Bolt project to the latest versions and best practices.
```

Comment:
The phrasing here seems a bit off. @hestonhoffman your thoughts? Maybe something along the lines of this?

```
Migrate a Bolt project to use current best practices and the latest versions of configuration files.
```

Reply:
Yeah, that sounds good